modules/persistent-storage-using-a-shared-configmap-in-a-pod.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: PROCEDURE
+
+[id="persistent-storage-using-a-shared-configmap-in-a-pod_{context}"]
```

Comment:
persistent-storage is actually misleading, as this is ephemeral storage ... can confuse future developers looking for content to contribute to

replace persistent with ephemeral

Reply:
@gabemontero Do you mean for this topic alone, or for all the topics in this PR? (They all use the `[id="persistent-storage` prefix.)

Reply:
I updated all the new files to begin with  `persistent-storage`